lib/fluent/plugin/in_cloudwatch_logs.rb

```diff
@@ -107,63 +110,80 @@ def run
         if Time.now > @next_fetch_time
           @next_fetch_time += @fetch_interval
 
-          if @use_log_stream_name_prefix || @use_todays_log_stream
-            log_stream_name_prefix = @use_todays_log_stream ? get_todays_date : @log_stream_name
-            begin
-              log_streams = describe_log_streams(log_stream_name_prefix)
-              log_streams.concat(describe_log_streams(get_yesterdays_date)) if @use_todays_log_stream
-              log_streams.each do |log_stream|
-                log_stream_name = log_stream.log_stream_name
-                events = get_events(log_stream_name)
-                events.each do |event|
-                  emit(log_stream_name, event)
+          if @use_log_group_name_prefix
+            log_groups = describe_log_groups(@log_group_name)
+          else
+            log_groups = [@log_group_name]
+          end
+
+          log_groups.each do |log_group|
+            log_group_name = log_group.log_group_name
```

Comment:
`log_group#log_group_name` is really existing?
`log_group_name = log_group` would work but is this line intended, @angel-manuel?

Reply:
I've changed that code. Now the array contains string, before that it could contain either strings or AWS SDK log group objects